packetbeat/protos/redis/redis_test.go

```diff
@@ -70,6 +70,24 @@ func TestRedisParser_ArrayRequest(t *testing.T) {
 	assert.Equal(t, len(arrayRequest), msg.size)
 }
 
+var arrayRequest2 = []byte("*3\r\n" +
+        "$6\r\n" +
```

Comment:
the extra whitespace here is probably what's breaking the linter... can you run `mage fmt update` and re-commit to make sure everything is formatted right?

Reply:
I get erros, would you please help me?

# command-line-arguments
./magefile.go:153:13: undefined: "github.com/elastic/beats/dev-tools/mage".DefaultIncludeListOptions
./magefile.go:156:39: not enough arguments in call to "github.com/elastic/beats/dev-tools/mage".GenerateIncludeListGo
Error: error compiling magefiles